src/pyhf/infer/calculators.py

```diff
@@ -22,12 +22,15 @@ def generate_asimov_data(asimov_mu, data, pdf, init_pars, par_bounds):
         pdf (~pyhf.pdf.Model): The statistical model adhering to the schema ``model.json``.
         init_pars (`tensor`): The initial parameter values to be used for fitting.
         par_bounds (`tensor`): The parameter value bounds to be used for fitting.
+        fixed_vals (`tensor`): Parameters to be held constant in the fit.
```

Comment:
We should probably harmonize this docstring across the whole PR. I don't have strong feelings one way or another on it.